src/utils/types.tsx

```diff
@@ -14,3 +14,10 @@ export type IconProps = {
 export type ByBreakpoint<T> = T | {mobile: T; tablet?: T; desktop: T};
 
 export type HeadingType = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'span';
+
+export const isBiggerHeading = (heading: HeadingType, otherHeading?: HeadingType): boolean => {
+    if (!otherHeading) {
+        return true;
+    }
+    return heading !== 'span' && (otherHeading === 'span' || heading < otherHeading);
```

Comment:
this means that if both headings are `'span'`, the second one wins?

Reply:
I'd add some early returns to simplify this boolean expression

Reply:
According to design specs, same heading level should never be used.

That being said, I think in case the headings are the same, it's better for the first one to "beat" the second one. I'll change it

Reply:
Done